### PR TITLE
Update/test-dpad-trigger

### PR DESCRIPTION
**Pull request by CrazyFloob**

### CODE REVIEW

File: src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+		when(controller.getPOV()).thenReturn(-1);
+        
+		DPadTrigger Dpad = new DPadTrigger(controller,triggerValue);
+
+		assertEquals(Dpad.get(), false);
+
+		Dpad.close();
+
+
```
Comment (Stick-2357) [inline src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java]: Remove excess space

File: src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		DPadValue triggerValue = DPadValue.fromPOV(1);
+
+		when(controller.getPOV()).thenReturn(1);
+        
+		DPadTrigger Dpad = new DPadTrigger(controller,triggerValue);
+
+		assertEquals(Dpad.get(), true);
+
+		Dpad.close();
+
```
Comment (Stick-2357) [inline src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java]: remove excess space

File: src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java (hunk 1 above)
Comment (NC2357) [reply src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java]: Done

File: src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java (hunk 2 above)
Comment (NC2357) [reply src/test/java/com/systemmeltdown/robotlib/triggers/DPadTriggerTest.java]: Done

**Pull request merged**